token-transfer-server/src/scripts/insert_ogv_airdrop_grants.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+const BigNumber = require('bignumber.js')
+const Logger = require('logplease')
+const moment = require('moment')
+
+const db = require('../models')
+
+const { getBalance } = require('../lib/balance')
+
+Logger.setLogLevel(process.env.LOG_LEVEL || 'INFO')
+const logger = Logger.create('enqueue', { showTimestamp: false })
```

Comment:
supernit: rename the logger to reflect the script name or equivalent. and use logger throughout the script vs a mix of logger and console.log.

Reply:
Oops 👍